WC_Pagantis.php

```diff
@@ -314,10 +315,38 @@ public function pagantisActivation()
         update_option('woocommerce_pagantis_settings', $settings);
     }
 
+    /**
+     * Product simulator PP o SELECTABLE
+     */
+    public function pagantisAddProductSimulatorPP()
```

Comment:
This method name is not very easy to read or understand, in my opinion a better name would be addProductSimulatorBeforeCartForm()

Reply:
Done